### PR TITLE
[INLONG-5321][DataProxy] Add zone test for MQ sinks

### PR DESCRIPTION
**Pull request by liangyepianzhou**
### Prepare a Pull Request
- Fixes https://github.com/apache/inlong/issues/5321

### Motivation

Now, the PulsarZoneSink, the KafkaZoneSink, and the TubeZoneSink have no test to verify. So we add a test to verify them.

### Modifications

Add test to verify zone test.

### CODE REVIEW

**Comment by liangyepianzhou** [comment]
@dockerzhang Could you please help review this PR when you have time? THX.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import org.apache.flume.Event;
+import org.apache.flume.Transaction;
+import org.apache.flume.channel.MemoryChannel;
+import org.apache.flume.event.EventBuilder;
+import org.apache.inlong.dataproxy.sink.kafkazone.KafkaZoneSink;
+import org.apache.inlong.dataproxy.sink.pulsarzone.PulsarZoneSink;
+import org.apache.inlong.dataproxy.sink.tubezone.TubeZoneSink;
+import org.junit.Test;
+
+public class TesZoneSink {
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java:31]: Is the `Tes` typo?

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        for (int i = 0; i < 10; i++) {
+            channel.put(event);
+        }
+        transaction.commit();
+        transaction.close();
+    }
+
+    @Test
+    public void testTubeZoneSinkProcess() {
+
```
Comment (healchow) [inline inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java:55]: Unnecessary blank lines.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java (hunk 1 above)
Comment (liangyepianzhou) [reply inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java]: Yeah, I didn't notice. Thanks for the reminder.

File: inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java (hunk 2 above)
Comment (liangyepianzhou) [reply inlong-dataproxy/dataproxy-source/src/test/java/org/apache/inlong/dataproxy/sink/zone/TesZoneSink.java]: Thanks for the reminder.

**Comment by liangyepianzhou** [comment]
@healchow  @dockerzhang  All tests have passed, please help review this PR, THX.

**Pull request closed without merging**